### DISCUSSION TITLE
where are the play positions for books saved?

### DISCUSSION THREAD

**Discussion by Shadesss** [PaulWoitaschek/Voice#1355] [Q&A]
Switched my sd card where I keep my audiobooks on to a bigger one. Now all the books are shown as "Not started yet".

Where are the play positions for the books saved? Maybe I could restore the positions and not have to manually do it for each book 😅

**Comment by gamingexpert13** [comment]
I have the exact same issue. Lost all my 'resume timestamps'. Sad to see this unanswered for almost a year.
I have another audiobook player that saves the positions in the folder with the audiobook, so i assumed this app would do something similar or at least recognize from the author/title/whatever that it's the exact same book.

**Comment by gamingexpert13** [comment]
Well i found the data folder for the app, but there's nothing in there but the cover images:
Android/data/de.ph1b.audiobook
